zowe-install/src/main/resources/component-scripts/start.sh

```diff
@@ -122,4 +122,5 @@ _BPX_JOBNAME=${ZOWE_PREFIX}${GATEWAY_CODE} java -Xms32m -Xmx256m -Xquickstart \
     -Dserver.ssl.trustStoreType=${KEYSTORE_TYPE} \
     -Dserver.ssl.trustStorePassword=${KEYSTORE_PASSWORD} \
     -Djava.protocol.handler.pkgs=com.ibm.crypto.provider \
+    -cp /usr/include/java_classes/IRRRacf.jar \
```

Comment:
We are using this file for "local" deployments on our mf system. I need to test if this is not breaking the way we are running.

Reply:
I've tested and it's running, on our test system, this file does not exist. How is this implementation accounting for differences in user environments? I feel that this is not the best way.

Reply:
although it's running, our thin jar is not finding this class, as it's using manifest classpath to find libraries

Reply:
even with fat jars, the -cp entry is not effective, i am still getting `cf.IRRPassTicket is not available, it will continue with default one org.zowe.apiml.passticket.PassTicketService$$Lambda$470/0000000`

Reply:
On what system?

Reply:
We have found how to configure the classpath correctly. The JZOS batch launcher is able to define CLASSPATH and use -jar at the same time but CLI java launcher does not. We have used just -cp and started the same class as the jar. That worked well. Since this is defined location by z/OS and Java products like MSM have used that for decade without a problem and required only JAVA_HOME to be configurable.

Reply:
The change has been tested on a Zowe system and a system deployed using David's zowe-api-dev deployment.